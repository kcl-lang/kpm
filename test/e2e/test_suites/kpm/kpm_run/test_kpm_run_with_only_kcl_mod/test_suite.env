KPM_HOME=""
KCLVM_VENDOR_HOME=""