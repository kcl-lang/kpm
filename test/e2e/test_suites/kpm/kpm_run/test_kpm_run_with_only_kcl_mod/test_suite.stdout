adding 'k8s' with version '1.27'
apiVersion: v1
kind: Pod
metadata:
  name: web-app
spec:
  containers:
  - image: nginx
    name: main-container
    ports:
    - containerPort: 80
